packages/xstate-wallet/src/workflows/tests/store.ts

```diff
@@ -40,4 +41,8 @@ export class TestStore extends XstateStore implements Store {
     if (!peerId) throw 'No peer';
     this.backend.setLedger(peerId?.participantId, channelId);
   }
+
+  async setPrivateKey(address: string, pk: string) {
+    await this.backend.setPrivateKey(address, pk);
+  }
```

Comment:
Can this calculate the address from the private key? Either that, or validate that the address is correct?